sybil/sybil.py

```diff
@@ -143,37 +154,37 @@ def parse(self, path: Path) -> Document:
         type_ = self.document_types.get(path.suffix, self.default_document_type)
         return type_.parse(str(path), *self.parsers, encoding=self.encoding)
 
-    def pytest(self):
+    def pytest(self) -> Callable[[Path, _PytestNodesCollectorType], Optional[_IntegrationPytestSybilFileType]]:
         """
         The helper method for when you use :ref:`pytest_integration`.
         """
         from .integration.pytest import pytest_integration
         return pytest_integration(self)
 
-    def unittest(self):
+    def unittest(self) -> Callable[[Optional[TestLoader], Optional[TestSuite], Optional[str]], TestSuite]:
         """
         The helper method for when you use :ref:`unitttest_integration`.
         """
         from .integration.unittest import unittest_integration
         return unittest_integration(self)
 
 
-class SybilCollection(list):
+class SybilCollection(List[Sybil]):
     """
     When :class:`Sybil` instances are concatenated, the collection returned can
     be used in the same way as a single :class:`Sybil`.
 
     This allows multiple configurations to be used in a single test run.
     """
 
-    def pytest(self):
+    def pytest(self) -> Callable[[Path, _PytestNodesCollectorType], Optional[_IntegrationPytestSybilFileType]]:
```

Comment:
```suggestion
    def pytest(self) -> Callable:
```

Reply:
Why do you prefer that to `Callable[[Path, Any], Any]`?

Reply:
Sorry, I misunderstood, `Callable[[Path, Any], Any]` is great, we just want to avoid links off to pytest and unittest internals that either cause dependency problems, or cause sphinx problems.